### ISSUE TITLE
The redirect_uri URL must be absolute!

### ISSUE THREAD

**Issue by kubilaytural** [adamjmcgrath/react-native-simple-auth#91]
Hello,

On iOS and android I get the following error.
"The redirect_uri URL must be absolute"

This error comes when the browser starts.

What should I do? Please help.

**Comment by adamjmcgrath** [comment]
Do you have a link to your project source code? I'd be interested to see what you set the `callback` value to in your provider. See https://github.com/adamjmcgrath/ReactNativeSimpleAuthExample/blob/master/secrets.example.js for examples

**Comment by kubilaytural** [comment]
Hi,

I solved my problem.
Add the codes below.

componentDidMount () {
Linking.addEventListener ('url', this.handleOpenURL);
Linking.getInitialURL ()
Then ((url) => {
if (url) {
console.log ('Starting URL:', url);
}
})
.catch (err => console.error ('An error occurred', err));
}

componentWillUnmount () {
Linking.removeEventListener ('url', this.handleOpenURL);
}

handleOpenURL (event) {
console.log ('handleOpenURL event.url', event.url);
}


The problem was resolved in this way.

**Comment by kubilaytural** [comment]
Hello there,

I solved my problem.
Add the codes below.


`
componentDidMount () {
Linking.addEventListener ('url', this.handleOpenURL);
Linking.getInitialURL ()
Then ((url) => {
if (url) {
console.log ('Starting URL:', url);
}
})
.catch (err => console.error ('An error occurred', err));
}

componentWillUnmount () {
Linking.removeEventListener ('url', this.handleOpenURL);
}

handleOpenURL (event) {
console.log ('handleOpenURL event.url', event.url);
}`


The problem was resolved in this way.
